functions/api/image/nfts/asset/[[index]].tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable import/no-unused-modules */
+import { ImageResponse } from '@vercel/og'
+import React from 'react'
+
+import getAsset from '../../../../utils/getAsset'
+import getSetup from '../../../../utils/getSetup'
+
+export async function onRequestGet({ params, request }) {
```

Comment:
Is this an external API? Why not implement it in utils, or is it _meant_ to be external?

Reply:
yup! its meant to be external